src/utils.ts

```diff
@@ -203,3 +203,36 @@ export function convertErrorToJson(err: Error) {
 
   return jsonObj;
 }
+
+/**
+ * isOnline safely return the navigator.online value
+ * if navigator is not in global object, it always return true
+ */
+export function isOnline() {
+  const nav =
+    typeof navigator !== 'undefined'
+      ? navigator
+      : typeof window !== 'undefined' && window.navigator
+      ? window.navigator
+      : undefined;
+
+  if (!nav) return true;
```

Comment:
Minor: but maybe we could add a warning if there's no navigator present, to alert users trying to use this in other envs.

Reply:
Same for the other methods in this module. Or alternatively, we could name the methods something else like (sBrowserConnected, isBrowserOnline... not sure about the naming though.

Reply:
warning is a good idea